website/docs/quickstart/approve.mdx

```diff
@@ -73,6 +74,75 @@ Total 3 (delta 2), reused 0 (delta 0), pack-reused 0
 remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
 ```
 
+## Using the CLI
+
+### Prerequisites
+
+- Proxy and REST API are running ([default behaviour](https://github.com/finos/git-proxy/blob/main/index.js))
+- The Git Proxy URL is configured via the GIT_PROXY_UI_HOST (defaults to `http://localhost`) and GIT_PROXY_UI_PORT (defaults to `8080`) environment variables. Note: this documentation assumes that Git Proxy UI is running on `http://git-proxy.com:8080`.
+- [Intercepting a push](/docs/quickstart/intercept) instructions have been followed and you've reached [Push via Git Proxy](/docs/quickstart/intercept#push-via-git-proxy)
+
+### Instructions
+
+#### 1. Find the tracking `ID`
+
+Following on from [Push via Git Proxy](/docs/quickstart/intercept#push-via-git-proxy), you'll receive a unique URL:
+
+```
+http://localhost:8080/requests/0000000000000000000000000000000000000000__79b4d8953cbc324bcc1eb53d6412ff89666c241f
+```
+
+The `ID` for your push corresponds to the last part of the URL:
+
+```
+0000000000000000000000000000000000000000__79b4d8953cbc324bcc1eb53d6412ff89666c241f
+```
+
+#### 2. Authenticate with the CLI
+
+Use the default & auto-generated Git Proxy username & password credentials to authenticate and obtain a cookie. The cookie value is saved to a file (`git-proxy-cookie`):
+
+```bash
+$ npx -- @finos/git-proxy-cli login --username admin --password admin
+Git-Proxy URL: http://git-proxy.com:8080
+Login 'admin': OK
+```
+
+#### 3. Approve the push with `ID`
+
+Use the commit `ID` to approve your push with the CLI:
+
+```bash
+$ npx -- @finos/git-proxy-cli approve --commitId 0000000000000000000000000000000000000000__79b4d8953cbc324bcc1eb53d6412ff89666c241f
+Git-Proxy URL: http://git-proxy.com:8080
```

Comment:
Is this the actual URL that will be displayed to the developer?

Reply:
leftover from previous commit, updated